src/lib.rs

```diff
@@ -474,6 +474,7 @@ macro_rules! try_opt {
     };
 }
 
+
```

Comment:
Spurious?

Reply:
just introduced some whitespace to get the CI running.. then realised there was a better way :D